pints/_log_pdfs.py

```diff
@@ -40,6 +40,13 @@ def evaluateS1(self, x):
         """
         raise NotImplementedError
 
+    def mean(self):
```

Comment:
I think this should be in `LogPrior`, not in `LogPDF`. In general we don't know the mean of the LogPDFs, that's the whole point :-)

Reply:
Also, we don't really expect most/many LogPDF classes to implement this method; having it on the interface makes it look too much like a requirement for me